gradle/libs.versions.toml

```diff
@@ -45,6 +45,7 @@ postgresql = "42.6.0"
 immutables-value = "2.10.0"
 selenium = "3.141.59"
 rauschig = "1.2.0"
+kyuubi = "1.8.0"
```

Comment:
I think you should also update the LICENSE.bin if possible.

Reply:
done